fabtests/unit/mr_cache_evict.c

```diff
@@ -437,14 +533,23 @@ static int mr_cache_test(enum alloc_type type)
 	/* A priming MR registration is used to ensure the first timed MR
 	 * registration does not take into account the setting up of CPU caches.
 	 */
-	prime_buf = malloc(mr_buf_size);
-	if (!prime_buf) {
-		ret = -ENOMEM;
-		FT_UNIT_STRERR(err_buf, "malloc failed", ret);
-		goto cleanup;
+	if (type == CUDA) {
```

Comment:
Use the iface comparison here.  The iface check more accurately captures what's driving the decision between using cuda_malloc vs malloc.

You would then have the iface to pass directly into the mr_register calls below to avoid the repeated conversion